.buildkite/custom-pipeline.yml

```diff
@@ -0,0 +1,52 @@
+steps:
+  - label: "build-gnu-x86-backend-stdio"
+    commands:
+      - cargo build --release --features=backend-stdio
+    retry:
+      automatic: false
+    agents:
+      platform: x86_64.metal
+      os: linux
+    plugins:
+      - docker#v3.0.1:
+          image: "rustvmm/dev:v6"
```

Comment:
Do you mind also fixing this: #34? We're using a really old container :(

Reply:
Sure, opened #37 for that.